MapboxNavigation/NavigationViewController.swift

```diff
@@ -684,12 +684,17 @@ extension NavigationViewController: NavigationServiceDelegate {
         
         if !isConnectedToCarPlay, // CarPlayManager shows rating on CarPlay if it's connected
             service.routeProgress.isFinalLeg && advancesToNextLeg && showsEndOfRouteFeedback {
-            self.mapViewController?.showEndOfRoute { _ in }
+            showEndOfRoute()
         }
         return advancesToNextLeg
 
     }
     
+    @objc public func showEndOfRoute(duration: TimeInterval = 1.0, completion: ((Bool) -> Void)? = nil) {
```

Comment:
2c349ab1a063e2acda1ccccb2589c002cf2bbc0c renames this method to `showEndOfRouteFeedback(duration:completionHandler:)` for consistency with the `showsEndOfRouteFeedback` property and Objective-C naming conventions.